packages/axelar-gmp-contract/src/evm-account-kit.js

```diff
@@ -0,0 +1,373 @@
+/**
+ * @import {VTransferIBCEvent} from '@agoric/vats';
+ * @import {Vow, VowTools} from '@agoric/vow';
+ * @import {Zone} from '@agoric/zone';
+ * @import {TypedPattern} from '@agoric/internal';
+ * @import {FungibleTokenPacketData} from '@agoric/cosmic-proto/ibc/applications/transfer/v2/packet.js';
+ * @import {ZoeTools} from '@agoric/orchestration/src/utils/zoe-tools.js';
+ * @import {AxelarGmpIncomingMemo, EvmTapState, ContractCall} from '../types';
+ * @import {ZCF, ZCFSeat} from '@agoric/zoe';
+ * @import {AxelarGmpOutgoingMemo} from '../types'
+ */
+
+/** @typedef {ContractCall} ContractCall */
+
+import { M, mustMatch } from '@endo/patterns';
+import { VowShape } from '@agoric/vow';
+import { makeTracer, NonNullish } from '@agoric/internal';
+import { atob, decodeBase64 } from '@endo/base64';
+import { decodeAbiParameters } from 'viem';
+import { Fail } from '@endo/errors';
+import { CosmosChainAddressShape } from '@agoric/orchestration';
+import { gmpAddresses, buildGMPPayload } from '../utils/gmp.js';
+
+const trace = makeTracer('EvmAccountKit');
+const { entries } = Object;
+
+const EVMI = M.interface('holder', {
+  getLocalAddress: M.call().returns(M.any()),
+  getAddress: M.call().returns(M.any()),
+  getLatestMessage: M.call().returns(M.any()),
+  send: M.call(M.any(), M.any()).returns(M.any()),
+  sendGmp: M.call(M.any(), M.any()).returns(M.any()),
+  fundLCA: M.call(M.any(), M.any()).returns(VowShape),
+});
+harden(EVMI);
+
+const InvitationMakerI = M.interface('invitationMaker', {
+  makeEVMTransactionInvitation: M.call(M.string(), M.array()).returns(M.any()),
+});
+harden(InvitationMakerI);
+
+/** @type {TypedPattern<EvmTapState>} */
+const EvmKitStateShape = {
+  localChainAddress: CosmosChainAddressShape,
+  sourceChannel: M.string(),
+  remoteDenom: M.string(),
+  localDenom: M.string(),
+  localAccount: M.remotable('OrchestrationAccount<{chainId:"agoric-3"}>'),
```

Comment:
`agoric-3` seems like it would be misleading on a localchain or testnet.

And for a debugging label, then length seems awkward.

```suggestion
  localAccount: M.remotable('LocalAccount'),
```

or `AgoricAccount`